src/Hammer.js

```diff
@@ -90,34 +91,33 @@ function updateHammer (hammer, props) {
 	});
 }
 
-var HammerComponent = React.createClass({
+class HammerComponent extends React.Component {
```

Comment:
We're not using babel, so using the new class syntax will break browser support. We need to use the `create-react-class` package instead and leave this as-is